sw/device/tests/BUILD

```diff
@@ -425,6 +425,7 @@ opentitan_test(
         "//hw/top_earlgrey:silicon_creator": None,
         "//hw/top_earlgrey:silicon_owner_sival_rom_ext": "silicon_owner",
         "//hw/top_earlgrey:silicon_owner_prodc_rom_ext": "silicon_owner",
+        "//hw/top_earlgrey:silicon_owner_proda_rom_ext": "silicon_owner",
```

Comment:
Do we really want to add `proda` to all the tests?

Reply:
In the past it has been useful to allow the silicon creator to test the sku before delivery. I think this could be restricted to just smoketests.